gr-blocks/CMakeLists.txt

```diff
@@ -54,11 +54,13 @@ add_subdirectory(lib)
 #  add_subdirectory(tests)
 #endif(ENABLE_TESTING)
 if(ENABLE_PYTHON)
-     add_subdirectory(python/blocks)
-     add_subdirectory(swig)
-     add_subdirectory(grc)
-     add_subdirectory(docs)
-     add_subdirectory(examples)
+    add_subdirectory(python/blocks)
+    add_subdirectory(swig)
+    if(ENABLE_GRC)
```

Comment:
I'd prefer that this ENABLE_GRC is outside the scope of ENABLE_PYTHON, to match all of the others.